New version: Perimeter81Ltd.Perimeter81Ltd version 10.4.3.1672

jnichols35 opened the pull request:
### Pull request has been created with [komac](https://github.com/russellbanks/Komac) v2.1.0 :rocket:

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/141459)

jnichols35 committed:
New version: Perimeter81Ltd.Perimeter81Ltd version 10.4.3.1672

stephengillie commented:
Hi @jnichols35,

This manifest adds Apps and Features entries that aren't present in previous PR versions. These entries should be added to the previous versions, or removed from this version.

(Automated response - build 855.)
<!--
[Policy] Needs-Author-Feedback
-->

jnichols35 committed:
Update Perimeter81Ltd.Perimeter81Ltd.installer.yaml

Removed:

```
AppsAndFeaturesEntries:
- UpgradeCode: '{81EB0138-5B33-4A31-9004-24FC73362241}'
```
